### PR TITLE
chore: move js examples, add dependabot, and update build workflow

### PR DESCRIPTION
**Pull request by poppoerika**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+<br>
+
+## Running the Example
+
+- Node version 16 or higher is required
+- A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+
+```bash
+cd javascript
```
Comment (pgautier404) [inline examples/README.md:13]:
```suggestion
cd examples
```

**Review by pgautier404 — COMMENTED** [review]
Left a small doc correction. This looks good, but I did get some kind of confusing output from the example:

```
[1663605139306] DEBUG (MomentoCache/5987 on momento): Request timeout: undefined
[1663605139306] DEBUG (MomentoCache/5987 on momento): Creating cache client using endpoint: 'cache.cell-external-beta-1.prod.a.momentohq.com'
[1663605139331] DEBUG (MomentoCache/5987 on momento): Request timeout: undefined
[1663605139331] DEBUG (MomentoCache/5987 on momento): Creating cache client using endpoint: 'cache.cell-external-beta-1.prod.a.momentohq.com'
[1663605139332] DEBUG (MomentoCache/5987 on momento): Request timeout: undefined
[1663605139332] DEBUG (MomentoCache/5987 on momento): Creating cache client using endpoint: 'cache.cell-external-beta-1.prod.a.momentohq.com'
[1663605139332] DEBUG (MomentoCache/5987 on momento): Request timeout: undefined
[1663605139332] DEBUG (MomentoCache/5987 on momento): Creating cache client using endpoint: 'cache.cell-external-beta-1.prod.a.momentohq.com'
[1663605139333] DEBUG (MomentoCache/5987 on momento): Request timeout: undefined
[1663605139333] DEBUG (MomentoCache/5987 on momento): Creating cache client using endpoint: 'cache.cell-external-beta-1.prod.a.momentohq.com'
[1663605139333] DEBUG (MomentoCache/5987 on momento): Request timeout: undefined
[1663605139333] DEBUG (MomentoCache/5987 on momento): Creating cache client using endpoint: 'cache.cell-external-beta-1.prod.a.momentohq.com'
[1663605139342] DEBUG (Momento/5987 on momento): Creating control client using endpoint: 'control.cell-external-beta-1.prod.a.momentohq.com
```

The example code seemed to execute just fine despite those debug messages, but it seems like they might be confusing to users.

**Comment by cprice404** [comment]
I think that if we rebase/merge from main on this PR now, and re-run the tests, hopefully they should pass now since the README template was fixed.

**Review by cprice404 — COMMENTED** [review]
hopefully tests will pass now, after rebase/update

**Comment by cprice404** [comment]
gah, no, my README thing is still slightly broken.  Need to post another fix for that.  Sorry.

File: examples/README.md (hunk 1 above)
Comment (poppoerika) [reply examples/README.md]: I realized that this `README` is already in `examples` directory, so I removed `cd` command altogether.

**Comment by poppoerika** [comment]
@pgautier404 
Regarding [this comment](https://github.com/momentohq/client-sdk-javascript/pull/101#pullrequestreview-1112548099), I agree that the current logging for the example code is a bit overwhelming. 
I changed the logging level to `INFO` instead of `DEBUG` so that it shows one logging output.
In this way, users can get a feel for our logging option.


```
{"level":30,"time":1663619975470,"pid":81279,"hostname":"Erikas-MacBook-Pro.local","name":"Momento","msg":"Creating cache: cache"}
cache already exists
Listing caches:
cache
localhost-3000-json
cache-test
default-cache
python-cache
users
Storing key=key, value=value, ttl=10
cache hit: value
success!!
```

**Review by cprice404 — COMMENTED** [review]
hey @poppoerika - I think the build is failing now because the build from the main directory is descending into the examples dir and getting confused.  I think we need to add something to tsconfig and/or package.json to cause it to exclude the `examples` subdir.  Hopefully this is locally reproducible via `npm run build`.

**Review by cprice404 — APPROVED** [review]

**Pull request merged**